cachito/workers/pkg_managers/gomod.py

```diff
@@ -208,8 +208,10 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
         if "force-gomod-tidy" in flags or dep_replacements:
             run_gomod_cmd(("go", "mod", "tidy"), run_params)
 
+        go_list = ["go", "list", "-e"]
```

Comment:
Could we also decide `-mod=vendor` up here already? It looks like the only command which is currently missing -mod=vendor is `go list -m`, and I think I've seen failures because of it

Reply:
Just checking--did you mean -mod=readonly here?

Reply:
Ah, right, yes I did :+1:

Reply:
Should we mention in commit message why we added "-mod", "readonly" flags?

Reply:
> Should we mention in commit message why we added "-mod", "readonly" flags?
Updated commit message :+1: